templates/beta/test/beta.js

```diff
@@ -1,406 +0,0 @@
-const { assertRevert } = require('@aragon/test-helpers/assertThrow')
```

Comment:
Are these tests supposed to be removed? Does the docker.js tests supersede these?

Reply:
Yes, that's the idea. Old `beta.js` was meant to be run on ganache, deploying everything from scratch. Now we use `scripts/deploy.sh` to test, which previously does all the needed deployment.